app/pb_kits/playbook/index.js

```diff
@@ -34,6 +34,7 @@ export HomeAddressStreet from './pb_home_address_street/_home_address_street.jsx
 export Icon from './pb_icon/_icon.jsx'
 export IconCircle from './pb_icon_circle/_icon_circle.jsx'
 export IconValue from './pb_icon_value/_icon_value.jsx'
+export IconStatValue from 'pb_icon_stat_value/_icon_stat_value.jsx'
```

Comment:
```suggestion
export IconStatValue from 'pb_icon_stat_value/_icon_stat_value.jsx'
export IconValue from './pb_icon_value/_icon_value.jsx'
```

These are not alpha ordered.